src/routes/recovery/recovery.controller.spec.ts

```diff
@@ -82,27 +102,137 @@ describe('Recovery (Unit)', () => {
       );
 
       await request(app.getHttpServer())
-        .post(`/v1/chains/${chainId}/safes/${safeAddress}/recovery`)
-        .send(addRecoveryModuleDto)
+        .post(`/v1/chains/${chain.chainId}/safes/${safe.address}/recovery`)
+        .set('Safe-Wallet-Signature', signature)
+        .set('Safe-Wallet-Signature-Timestamp', timestamp.toString())
+        .send({
+          ...addRecoveryModuleDto,
+          // TODO: Add to request
+          signer: signer.address,
+        })
         .expect(200);
     });
 
-    it('should get a validation error', async () => {
+    it('should prevent requests older than 5 minutes', async () => {
+      const addRecoveryModuleDto = addRecoveryModuleDtoBuilder().build();
+      const privateKey = generatePrivateKey();
+      const signer = privateKeyToAccount(privateKey);
+      const chain = chainBuilder().build();
+      const safe = safeBuilder().with('owners', [signer.address]).build();
+      const timestamp = jest.now() - (5 * 60 * 1000 + 1); // 5m + 1ms
```

Comment:
Not really something that needs to be changed but other tests advance the timers instead of signing an older timestamp. https://github.com/safe-global/safe-client-gateway/blob/97c3188a1d30a313227b9d5c664ff92bdc61b59d/src/routes/email/email.controller.save-email.spec.ts#L184

Both should be valid.

Reply:
Changed in cab30bf4823a04b278ad156652bfe0b319df3044.